src/_help/changes-management/integrations.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Integrations
+---
+
+- TOC
+{:toc}
+
+## Slack integration
+
+Integrating Bump.sh with Slack can be done in three steps.
+
+1. From your Slack workspace admin, [create a webhook](https://slack.com/apps/A0F7XDUAZ-incoming-webhooks): this allows you to specify where Bump.sh should send change notifications.
+2. Once you have copied the webhook URL, go to the settings of your documentation, in the Integrations tab, enable the Slack option, and provide the previously obtained webhook URL.
+3. Mention the channels where you want to receive notifications.
+
+After configuration, remember to test the proper reception of notifications with the test button.
+
+![](/images/help/slack-integration.png)
+
+## Webhooks
+
+You can define as many webhooks as you wish to receive structural changes when they occur on your documentations.
+
+A webhook is a way for an app to get instant notifications from other applications. Webhooks are automated messages (also called payloads) sent to a specific URL. Unlike a classic call to an API, webhooks are more efficient than just polling it.
+
+Webhooks help integrate Bump into your current workflows. The chosen URLs will receive an HTTP request automatically when Bump notices a structural change in your documentation.
+
+Notifications can be sent to several URLs, allowing better integration with your existing tools.
```

Comment:
Same here, we should improve the description here to add concrete examples of why using webhooks: update an API client generator, re-build an integration or security test suite, etc...

Reply:
I'm gonna use the ones you shared here as I don't have an exhaustive list of all the possibility behind webhooks

Reply:
A much simpler proposal for the whole paragraph (from "You can define as"  to "with your existing tools"):

> When your API evolves, you may want to automatically launch some related tasks: re-build your API client, update your security test suite, your integration tests, etc... This can be done thanks to Bump.sh webhooks.

> It will send a specific request to the URLs you have configured each time it detects a change in your API structure. You can define as many webhooks as you wish.